.travis.yml

```diff
@@ -12,6 +12,8 @@ matrix:
     jdk: oraclejdk9
   - scala: 2.12.6
     jdk: oraclejdk10
+  - scala: 2.12.7
+    jdk: oraclejdk11
```

Comment:
We’re going to support only OpenJDK since JDK 11. Could you change this to openjdk11?